### PR TITLE
[onert] add shape inference for ReduceMin op and its test case.

### PR DESCRIPTION
**Pull request by Hyunjun85**
Add shape inference for ReduceMin op and its test case.

Isssue : https://github.com/Samsung/ONE/issues/2281

ONE-DCO-1.0-Signed-off-by: H Kim <hyunjun2.kim@samsung.com>

### CODE REVIEW

File: runtime/onert/core/src/util/shapeinf/ReduceMin.cc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+void DynamicInferer::visit(const ir::operation::ReduceMin &op)
+{
+  const auto input_idx{op.getInputs().at(0)};
+  const auto &input = _tensor_registry->getITensor(input_idx);
+  auto input_shape = getShape(input.get());
+
+  if (!input->is_dynamic())
+    return;
+
+  const auto axes = op.param().axes;
```
Comment (ragmani) [inline runtime/onert/core/src/util/shapeinf/ReduceMin.cc:56]: `axis` can be nagetive number. Could you add applying it?

Comment (Hyunjun85) [reply runtime/onert/core/src/util/shapeinf/ReduceMin.cc:56]: The negative value of axis will handle in ```[ShapeInference.cc::inferRecudeShapes()](https://github.com/Samsung/ONE/blob/master/runtime/onert/core/src/util/shapeinf/ShapeInference.cc#L124)```.
My think, It is not problem.

**Comment by Hyunjun85** [comment]
Resolve conflict and rebase from master.

File: runtime/onert/core/src/util/shapeinf/ReduceMin.cc (hunk 1 above)
Comment (ragmani) [reply runtime/onert/core/src/util/shapeinf/ReduceMin.cc:56]: I haven't seen that part. Thanks.

**Comment by Hyunjun85** [comment]
https://github.com/Samsung/ONE/issues/2281

**Pull request merged**